### ISSUE TITLE
Karma system

### ISSUE THREAD

**Issue by juanjp600** [Regalis11/Barotrauma#67]
Basically, the idea is to penalize players that harm players, damage the sub and destroy equipment. If a player has been doing a lot of harm, their skills, health and clearance should be reduced to the point where they can't do anything.

**Comment by Crystalwarrior** [comment]
That's a good idea, though definitely let server hosts tweak it up to severity. Will definitely help with griefers as well.
Perhaps keep track of "casualties" because of your actions? Like if you sank the sub into the abyss because of cutting a hole, your karma would plummet just as fast

**Comment by juanjp600** [comment]
Yeah, obviously hosts should have the ability to tweak or disable the karma system.
Keeping track of the consequences could be possible to some extent, though you can't really go too far with it because it may be inaccurate after some time.
